broadway/api/utils/multiqueue.py

```diff
@@ -0,0 +1,45 @@
+from queue import Queue, Empty
+
+"""
+A wrapper around Python's queue module that contains multiple queues
+and round-robins between them when pulling.
+"""
+
+
+class MultiQueue:
+    def __init__(self):
+        self.queues = {}
+        self.keys = []
+        self.round_robin_idx = 0
+
+    def add_queue(self, queue_id):
```

Comment:
We can make this a private method since this does not need to be used externally?
(we can make it private by naming it `_add_queue`)